ui/utils/Enum.js

```diff
@@ -55,6 +55,6 @@ export const MesheryPatternsCatalog = 'meshery-patterns-catalog';
 export const MesheryFiltersCatalog = 'meshery-filters-catalog';
 
 export const CONNECTION_KINDS = {
-  MESHERY: 'meshery-core',
```

Comment:
@senali-d you are changing to meshery but above you are adding conditional logic to change to meshery-core. The real issue I think is because at https://github.com/meshery/meshery/pull/9538/files#diff-b14dd66531b84fdbb46ba6c386362eb972e04bf8d3abe3b6f26e95a6c16fa493L236
splitting logic. Makes sense.

thanks for handling  the issue though.